index.html

```diff
@@ -1,26 +1,25 @@
 <head>
   <link rel="stylesheet" href="./style.css" type="text/css">
   <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-</head>
 <body>
-<div class = "all">
-  <h3 class="">Simon</h3>
-  <div class="row">
-    <div>
-      <button class="btn btn-default start clickable" id="Start">Start</button>
-    </div>
+  <div class = "container">
+    <h3 class="">Simon</h3>
 
-  </div>
-  <div class="buttons" id="buttons">
-    <div class="row">
-      <button class="btn btn-default target green 1" id="Green">Green</button>
-      <button class="btn btn-default target red 2" id="Red">Red</button>
-    </div>
-    <div class="row">
-      <button class="btn btn-default target yellow 3" id="Yellow">Yellow</button>
-      <button class="btn btn-default target blue 0" id="Blue">Blue</button>
+    <div class="buttons container-fluid" id="buttons">
+      <div class="row">
+        <button class="target green 1" id="Green"></button>
```

Comment:
class identifiers may not begin with an integer, so `1` is an invalid class identifier here.  As well, standard practice is to use lower kebab case (`green-thing-whatevs`)  in class names.  Finally, best practice is to avoid the use of `id`s in HTML tags.